forum/users/serializers.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework import serializers
+from rest_framework.validators import UniqueValidator
+
+from users.models import CustomUser
+from users.validators import CustomUserValidator
+
+class UserRegisterSerializer(serializers.ModelSerializer):
+    password = serializers.CharField(write_only=True, required=True)
+    password2 = serializers.CharField(write_only=True, required=True)
+
```

Comment:
Maybe it should be useful for any email unique validator?
```
email = serializers.EmailField(
        required=True,
        validators=[UniqueValidator(queryset=CustomUser.objects.all())]
    )
```

Reply:
Added, thanks